zenodo/modules/deposit/static/json/zenodo_deposit/deposit_form.json

```diff
@@ -287,6 +287,7 @@
       {
         "type": "textarea",
         "key": "notes",
+        "title": "Additional Notes",
```

Comment:
We should normally strive to use UK english spelling i.e. lowercase n ``Additional notes``. (I know it's not consistently applied).